### PR TITLE
Update New-NetQosPolicy.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the cluster description

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3110

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: docset/winserver2012r2-ps/netqos/New-NetQosPolicy.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ Accept wildcard characters: False
 ```
 
 ### -Cluster
-Specifies Microsoft's cluster service, which uses the default SQL server TCP/UDP port 3343.
+Specifies the Microsoft Windows Cluster Service, which uses the Cluster network driver TCP/UDP port 3343.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/netqos/New-NetQosPolicy.md:265]:
```suggestion
Specifies the Microsoft Windows Cluster service, which uses the Cluster network driver TCP/UDP port 3343.
```

File: docset/winserver2016-ps/netqos/New-NetQosPolicy.md [hunk 2]
```diff
@@ -262,7 +262,7 @@ Accept wildcard characters: False
 ```
 
 ### -Cluster
-Specifies the Microsoft cluster service, which uses the default SQL server TCP/UDP port 3343.
+Specifies the Microsoft Windows Cluster Service, which uses the Cluster network driver TCP/UDP port 3343.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/netqos/New-NetQosPolicy.md:265]:
```suggestion
Specifies the Microsoft Windows Cluster service, which uses the Cluster network driver TCP/UDP port 3343.
```

File: docset/winserver2019-ps/netqos/New-NetQosPolicy.md [hunk 3]
```diff
@@ -262,7 +262,7 @@ Accept wildcard characters: False
 ```
 
 ### -Cluster
-Specifies the Microsoft cluster service, which uses the default SQL server TCP/UDP port 3343.
+Specifies the Microsoft Windows Cluster Service, which uses the Cluster network driver TCP/UDP port 3343.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/netqos/New-NetQosPolicy.md:265]:
```suggestion
Specifies the Microsoft Windows Cluster service, which uses the Cluster network driver TCP/UDP port 3343.
```

File: docset/winserver2022-ps/netqos/New-NetQosPolicy.md [hunk 4]
```diff
@@ -262,7 +262,7 @@ Accept wildcard characters: False
 ```
 
 ### -Cluster
-Specifies the Microsoft cluster service, which uses the default SQL server TCP/UDP port 3343.
+Specifies the Microsoft Windows Cluster Service, which uses the Cluster network driver TCP/UDP port 3343.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/netqos/New-NetQosPolicy.md:265]:
```suggestion
Specifies the Microsoft Windows Cluster service, which uses the Cluster network driver TCP/UDP port 3343.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**